src/main/java/com/dynatrace/openkit/core/SessionImpl.java

```diff
@@ -16,15 +16,13 @@
 
 package com.dynatrace.openkit.core;
 
-import com.dynatrace.openkit.api.Action;
-import com.dynatrace.openkit.api.Logger;
-import com.dynatrace.openkit.api.RootAction;
-import com.dynatrace.openkit.api.Session;
+import com.dynatrace.openkit.api.*;

```

Comment:
can this pollute the imports, referencing stuff that was not intended? I would prefer 4 explicitly defined imports

Reply:
Basically, it's 5 imports, which is the default threshold for IntelliJ to use wildcard imports.

I've changed this in my IDE.